_cucumber/cucumber.rake

```diff
@@ -27,7 +27,13 @@ task :features do |t, args|
     profile = 'parallel'
   end
 
-  exit_status = runner.run(profile)
+  if ENV['CUCUMBER_TAGS'].to_s.empty?
+    tags = nil
+  else
+    tags = ENV['CUCUMBER_TAGS']
```

Comment:
I think it might be worth having some validation here. The only tags allowed are @smoke and @acceptance right?